pandas/tests/extension/test_arrow.py

```diff
@@ -0,0 +1,198 @@
+"""
+This file contains a minimal set of tests for compliance with the extension
+array interface test suite, and should contain no other tests.
+The test suite for the full functionality of the array is located in
+`pandas/tests/arrays/`.
+The tests in this file are inherited from the BaseExtensionTests, and only
+minimal tweaks should be applied to get the tests passing (by overwriting a
+parent method).
+Additional tests should either be added to one of the BaseExtensionTests
+classes (if they are relevant for the extension interface for all dtypes), or
+be added to the array-specific tests in `pandas/tests/arrays/`.
+"""
+
+from datetime import (
+    date,
+    datetime,
+    time,
+    timedelta,
+)
+
+import numpy as np
+import pytest
+
+from pandas.compat import (
+    pa_version_under2p0,
+    pa_version_under3p0,
+)
+
+import pandas as pd
+import pandas._testing as tm
+from pandas.tests.extension import base
+
+pa = pytest.importorskip("pyarrow", minversion="1.0.1")
+
+from pandas.core.arrays.arrow.dtype import ArrowDtype  # isort:skip
+
+
+@pytest.fixture(params=tm.ALL_PYARROW_DTYPES)
+def dtype(request):
+    return ArrowDtype(pyarrow_dtype=request.param)
+
+
+@pytest.fixture
+def data(dtype):
+    pa_dtype = dtype.pyarrow_dtype
+    if pa.types.is_boolean(pa_dtype):
+        data = [True, None, False, None, False, None]
```

Comment:
So related to my other comment, you can change the order of those values slightly so that it starts with `True, False`, and doesn't end with `None` (that will be needed for quite some other tests in future PRs as well)